### PR TITLE
Hcb badges

### PR DESCRIPTION
**Pull request by kimani-ndegwa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ __pycache__/
 hc.sqlite
 hc/local_settings.py
 static-collected
-
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+venv
```
Comment (iamlordaubrey) [inline .gitignore]: Any reason why you're having two virtual environment folders in your application?

**Review by iamlordaubrey — CHANGES_REQUESTED** [review]
I wouldn't like to clutter the .gitignore file with multiple virtual env folders. We should all have one called `hc-venv` (as per the readme).

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # healthchecks
 
+[![Code Climate](https://codeclimate.com/github/andela/healthchecks_b/badges/gpa.svg)](https://codeclimate.com/github/andela/healthchecks_b)
+[![Test Coverage](https://codeclimate.com/github/andela/healthchecks_b/badges/coverage.svg)](https://codeclimate.com/github/andela/healthchecks_b/coverage)
+[![Issue Count](https://codeclimate.com/github/andela/healthchecks_b/badges/issue_count.svg)](https://codeclimate.com/github/andela/healthchecks_b)
+[![PyPI](https://img.shields.io/pypi/pyversions/Django.svg?maxAge=2592000)]()
```
Comment (iamlordaubrey) [inline README.md]: While you're it, I'm not sure the python badge is necessary. If you can defend it's necessity, by all means please do so. Else, let's remove it. LESS is MORE 😁

File: .gitignore (hunk 1 above)
Comment (kimani-ndegwa) [reply .gitignore]: Thats actually whitney who had **venv** as her virtual environment. I will let her know.

File: README.md (hunk 2 above)
Comment (kimani-ndegwa) [reply README.md]: Ok. Noted as well.

File: .gitignore [hunk 3]
```diff
@@ -4,6 +4,6 @@ __pycache__/
 hc.sqlite
 hc/local_settings.py
 static-collected
-
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+hc_venv
```
Comment (iamlordaubrey) [inline .gitignore]: virtual env should be hc-venv not hc_venv. Please take note... a hyphen, not an underscore.

**Review by iamlordaubrey — APPROVED** [review]

**Pull request merged**